src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -150,9 +153,22 @@ extension RiskProvider: RiskProviding {
 	}
 	#else
 	private func _requestRiskLevel(userInitiated: Bool, completion: Completion? = nil) {
```

Comment:
consistency? Please correct me if I'm wrong, but since when do we underscore private methods?

Reply:
![image](https://media.giphy.com/media/S1ZnyFKKJ9rwc/giphy.gif)